test/module/irohad/network/block_loader_test.cpp

```diff
@@ -229,12 +231,13 @@ TEST_F(BlockLoaderTest, ValidWhenMultipleBlocks) {
 
   setPeerQuery();
   auto retrieved_blocks = loader->retrieveBlocks(1, peer_key).assumeValue();
-  auto wrapper = make_test_subscriber<CallExact>(retrieved_blocks, num_blocks);
+  size_t count = 0;
   auto height = next_height;
-  wrapper.subscribe(
-      [&height](auto block) { ASSERT_EQ(block->height(), height++); });
-
-  ASSERT_TRUE(wrapper.validate());
+  for (auto &block : retrieved_blocks) {
+    ++count;
+    ASSERT_EQ(block->height(), height++);
```

Comment:
```suggestion
    ASSERT_EQ(block->height(), height);
    ++height;
```
To be more expressive.